Tests/Integration/Core/OrderHelperTest.php

```diff
@@ -0,0 +1,256 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+use OxidEsales\Eshop\Application\Model\User;
+use OxidEsales\Eshop\Core\Registry;
+
+use Wirecard\Oxid\Core\OrderHelper;
+
+class OrderHelperTest extends \Wirecard\Test\WdUnitTestCase
+{
+    protected function dbData()
+    {
+        return [
+            [
```

Comment:
Yes, it would be a cleaner solution.

However, all other DB test data has been created & approved in a similar way. 

For this reason, we have created a separate ticket for it: https://jira.parkside.at/browse/WDPD-238

Reply:
As discussed in yesterday's approval, this is only needed in case the DB data setup is very complex and makes the test code less readable.

Therefore, we will address it in this PR right away. 👍